statsd/statsd_test.go

```diff
@@ -434,3 +434,12 @@ func TestEntityID(t *testing.T) {
 		t.Errorf("Expecting empty default tags, got %v", client.Tags)
 	}
 }
+
+func TestClosePanic(t *testing.T) {
+	c, err := statsd.New("localhost:8125")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Nit: While legacy test are using the Go's raw test api, new tests should use testify
```go
assert.Nil(t, err)
```

Reply:
Sorry for jumping in uninvited, another option would also be:
```
assert.NoError(t, err)
```

Reply:
Nobody's uninvited <3